### ISSUE TITLE
Bluetooth phone issue with DS4W

### ISSUE THREAD

**Issue by vpontin** [Ryochan7/DS4Windows#323]
Hi!

After updating my windows 10 to 1803 i began experiencing issues when trying to connect my bluetooth headphone. When turning on (already paired) my PC audio simply froze, and i need to turn off and turn on the bluetooth in the action center.

Back then i thought it was something related with the update and the audio drivers (conextant) or bluetooth (toshiba), BUT after formatting and reinstalling my PC the problem was gone!... until i installed again DS4Windows.

Now i'm using the ViGem build, but i have a very similar issue: when power on my headphone it wont connect immediately with PC, i need to turn off and on bluetooth to have audio (something i dont need to do before installing DS4W.

Also for noting: My laptop is a Lenovo G50, DS4W isnt running when this happens, and i checked hid ds4 controller in the options.

So, TL:DR: I have some connection issues bluetooth headset with different installations, both after having 1803 and DS4Windows installed. What i need to do?
